synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -0,0 +1,261 @@
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 2 of the License, or
+**	(at your option) any later version.
+**
+**	Synfig is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === H E A D E R S ======================================================= */
+
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include <gui/dialogs/dialog_guide.h>
+
+#include <gui/localization.h>
+
+#include <gtkmm/box.h>
+#include <gtkmm/frame.h>
+#include <gtkmm/grid.h>
+
+#include <gui/workarea.h>
+#include <gui/duckmatic.h>
+#include <gui/app.h>
+
+#endif
+
+using namespace synfig;
+using namespace studio;
+
+/* === M A C R O S ========================================================= */
+
+/* === G L O B A L S ======================================================= */
+/* === P R O C E D U R E S ================================================= */
+
+/* === M E T H O D S ======================================================= */
+
+Dialog_Guide::Dialog_Guide(Gtk::Window& parent, etl::handle<synfig::Canvas> canvas, WorkArea* work_area):
+	Dialog(_("Guide Editor"),parent),
+	canvas(canvas),
+	current_work_area(work_area),
+	angle_adjustment(Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+/*	center_x_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+	center_y_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+	point_x_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)),
+	point_y_widget_adjust (Gtk::Adjustment::create(0,-2000000000,2000000000,1,1,0)), */
+	degrees(true)
+{
+	this->set_resizable(false);
+	assert(canvas);
+
+	//Box start
+	Gtk::Box *guide_box = Gtk::manage(new Gtk::Box(Gtk::ORIENTATION_VERTICAL));
+
+	angle_widget=manage(new Gtk::SpinButton(angle_adjustment,15,2));
+	angle_widget->show();
+
+/*	center_x_widget=manage(new Gtk::SpinButton(center_x_widget_adjust,15,2));
+	center_x_widget->show();
+	center_x_widget->set_hexpand();
+	center_y_widget=manage(new Gtk::SpinButton(center_y_widget_adjust,15,2));
+	center_y_widget->show();
+	center_y_widget->set_hexpand();
+	point_x_widget=manage(new Gtk::SpinButton(point_x_widget_adjust,15,2));
+	point_x_widget->show();
+	point_x_widget->set_hexpand();
+	point_y_widget=manage(new Gtk::SpinButton(point_y_widget_adjust,15,2));
+	point_y_widget->show();
+	point_y_widget->set_hexpand(); */
+
+	Gtk::Frame* angleFrame = manage(new Gtk::Frame(_("Rotate By Setting an Angle")));
```

Comment:
"Rotate Guide"? I think it will be clearer for translators

Reply:
It was originally written this way because there was also an option to rotate through setting the two points that defined the guide. but now the guide is only defined by a pivot and an angle so I guess it could just be "Rotate Guide" then ok.

Reply:
> or would we add setting the guide in the dialog through using two points in the future so then it should be kept as is I'm not sure.

In this case, we'll just rename it back. Perhaps there will be a more understandable name.

P.S. What I mean is that translators only see the phrase to be translated, but they don't see the whole context. So more information might be helpful.

Reply:
Ok then I guess for now Rotate Guide would be sufficient.

Reply:
done